foundation/src/nativeMain/kotlin/com/mirego/trikot/foundation/date/Date.kt

```diff
@@ -20,6 +20,8 @@ actual class Date(val nsDate: NSDate) {
     }
 
     actual companion object DateFactory {
+        private const val epochReferenceDateDelta: Double = 978307200.0
```

Comment:
I think we should expose the formula to this number.

Reply:
I didn't use a formula, I simply output the result of:
```
let refDate = Date(timeIntervalSinceReferenceDate: 0)
let epochDate = Date(timeIntervalSince1970: 0)
refDate.timeIntervalSince(epochDate)
```
Not having access to `Date(timeIntervalSince1970:)` makes it a bit hard to express it clearly in code.